api/envoy/extensions/compression/gzip/decompressor/v3/gzip.proto

```diff
@@ -27,4 +27,10 @@ message Gzip {
   // Value for zlib's decompressor output buffer. If not set, defaults to 4096.
   // See https://www.zlib.net/manual.html for more details.
   google.protobuf.UInt32Value chunk_size = 2 [(validate.rules).uint32 = {lte: 65536 gte: 4096}];
+
+  // How many times the output buffer is allowed to be bigger than the size of
+  // accumulated input. This value is used to detect compression bombs.
+  // TODO(rojkov): Re-design the Decompressor interface to handle compression
+  // bombs gracefully instead of this quick solution.
```

Comment:
This should be inside a `[#comment`.

Also what is the default value of this field?

Also, how does this field relate to `chunk_size` that sets the output buffer size IIUC?

Reply:
The default value of this filed is 100 which be set in config.cc.

Reply:
`chunk_size` is used for zlib internal output buffer which controls the frequency of output copying. 
And `max_inflate_ratio` is in gzip decompressor level and limits the size of output buffer of `decompressor filter`.